warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

```diff
@@ -59,6 +59,10 @@ def open_browser(self, browser_name='firefox', webdriver_remote_url=False,
         browser_name = browser_name
         browser = self._make_browser(browser_name, desired_capabilities,
                                      profile_dir, webdriver_remote_url)
+        print_info("The Selenium Webdriver version is '{0}'".format(webdriver.__version__))
```

Comment:
Hi @brettleehari,

Please make this into a separate function (may be ```get_browser_specifications()```) so that:
1. We will be able to call it from anywhere
2. If we need to print out more details about the browser and/or selenium later, we have the room to keep on adding to that function

Thanks.